### PR TITLE
Added Share This buttons to event page

### PR DESCRIPTION
**Pull request by vladigor**

### CODE REVIEW

**Comment by ahaith** [comment]
We need to do something to populate an image (OU logo) and text for the preview. I presume the ShareThis service supports doing that somehow?

On Facebook at least, it looks a bit naff atm. We'll need to check that it comes out looking OK on the other channels too.

screen shot 2015-08-11 at 10 23 27

**Comment by vladigor** [comment]
The snippets are retrieved from the opengraph metadata. I've updated the opengraph meta to include the logo.
RE .facebook and linked in : I suspect that the problem of metadata not pulling through is caused by the fact that our dev machines aren't publicly accessible.  Can we push to dev and then test from there?   
The metadata for 'email' and 'sharethis', which is previewed directly in the page is displaying correctly.

**Comment by ahaith** [comment]
Deploying to dev server so we can test social links there.

File: talks/templates/events/event.html [hunk 1]
```diff
@@ -31,6 +32,11 @@
 </script>
 {% endif %}
 
+{# For Share This links #}
+<script type="text/javascript">var switchTo5x=true;</script>
+<script type="text/javascript" src="http://w.sharethis.com/button/buttons.js"></script>
```
Comment (ahaith) [inline talks/templates/events/event.html]: Make this src protocol-relative. The dev server refuses to load it, as it's from https.

**Comment by ahaith** [comment]
Screenshot from Twitter. No idea what's 'normal' for a tweet though!

screen shot 2015-08-12 at 15 10 16

**Pull request merged**